lib/src/model/model_function.dart

```diff
@@ -14,14 +14,11 @@ class ModelFunction extends ModelFunctionTyped with Categorization {
 
   @override
   bool get isStatic {
-    return _func.isStatic;
+    return (element as FunctionElement).isStatic;
```

Comment:
optional: one thing I've tried to do elsewhere to reduce the amount of casting is to override the `element` getter so it does the casting (and that way, the type is consistent)

Reply:
Ah ha, very nice. Done.